drivers/aks/aks_driver.go

```diff
@@ -752,10 +896,16 @@ func (d *Driver) createOrUpdate(ctx context.Context, options *types.DriverOption
 			driverState.Subnet,
 		))
 
-		networkProfile = &containerservice.NetworkProfile{
+		loadBalancerSku := containerservice.Basic
+		if driverState.LoadBalancerSku != "" {
```

Comment:
If we add this code block here, it means only users select `Advanced` Network mode then they can choose the type of loadBalancerSku. However, in Azure console, users can use Standard LB with default Network mode.

So we might need to support Standard LB with default Network in Rancher as well

Reply:
Resolved :+1: